### DISCUSSION TITLE
Can't authenticate to admin GUI with superadmin certificate

### DISCUSSION THREAD

**Discussion by foodani** [Keyfactor/ejbca-ce#501] [Q&A] [answered]
Hello,
I have deployed EJBCA version 8 on WildFly version 26.1.3 with Java 17 and MySQL 8, and it was successful without any errors. 
I then generated the CA and the superadmin.p12 certificate, imported it into the browser but I keep getting the 'Authorization Denied' error on the admin web page. On the WildFly logs I see 'Client certificate or OAuth bearer token required'. 
I followed the installation guide, I even tried regenerating the superadmin.p12 certificate but still can't authenticate.
Does anyone have any suggestions on what I can try?

Thank you
Dan

**Comment by primetomas** [comment, marked as answer]
If you are sure to have the right certificate installed in the web browser, restarted your browser (so no cached session gets in the way), and are accessing using port 8443. Then the most likely is that there was something wrong when you configured the HTTPS/TLS layer during installation. https://doc.primekey.com/ejbca/ejbca-installation/application-servers/wildfly-26#WildFly26-HTTP(S)Configuration

Port 8443 must be configured to require client certificate, from your Management CA (same CA as issued superadmin.p12), in which case it will ask you for client certificate and your web browser should prompt you for it.
